### PR TITLE
vmware/test: add vsphere-automation-sdk dep

### PR DESCRIPTION
**Pull request by goneri**
##### SUMMARY

The following modules depend on `vSphere Automation SDK`:

- `vmware_rest_client`
- `vmware_guest_info`
- `vmware_tag_manager`
- `vmware_vm_inventory`

The associated test cannot be run with `govcsim`. But the situation is
changing since we will soon run them on a regular lab, and so, we
need to install the dependency.

Depends-On: https://github.com/ansible/default-test-container/pull/31
Depends-On: https://github.com/ansible/ansible/pull/62285
Depends-On: https://github.com/ansible/ansible/pull/62412

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

vmware_rest_client
vmware_guest_info
vmware_tag_manager
vmware_vm_inventory

### CODE REVIEW

File: test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 pyvmomi
+git+https://github.com/vmware/vsphere-automation-sdk-python.git
```
Comment (mattclay) [inline test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt:2]: Is this SDK not available on pypi?

**Comment by goneri** [comment]
- The image comes with `pip` 19.0.2, we need 19.2.3 instead to be able to install the SDK.
- The SDK depends on `requests>=2.21.0` but `hcloud` 1.4.0 has requirement `requests==2.20.0`

File: test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt (hunk 1 above)
Comment (goneri) [reply test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt:2]: Nah, I asked myself the very same question: https://github.com/vmware/vsphere-automation-sdk-python/issues/38

Comment (Akasurde) [reply test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt:2]: No this is not available on PyPi :(

**Comment by goneri** [comment]
@mattclay this time, the CI is passing.

File: test/lib/ansible_test/_data/completion/docker.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-default name=quay.io/ansible/default-test-container:1.9.2 python=3.6,2.6,2.7,3.5,3.7,3.8 seccomp=unconfined
+default name=quay.io/ansible/default-test-container:1.9.3 python=3.6,2.6,2.7,3.5,3.7,3.8 seccomp=unconfined
```
Comment (mattclay) [inline test/lib/ansible_test/_data/completion/docker.txt]: Can you split these two updates into separate PRs? We'll merge the default test container change first. We may end up only backporting the default container update.
